akka-docs/src/main/paradox/java/remoting-artery.md

```diff
@@ -141,10 +149,21 @@ In the next sections the two alternatives are described in detail.
 
 `actorSelection(path)` will obtain an `ActorSelection` to an Actor on a remote node, e.g.:
 
-```
-ActorSelection selection =
-  context.actorSelection("akka://actorSystemName@10.0.0.1:25520/user/actorName");
-```
+Scala
+:   @@@vars
```

Comment:
Is it not only for this place but generally for in-line code snippets **_not_** using `${property}` values? 

If we should remove unnecessary `@@@vars`  from in-line snippets, I can create a separate PR updating **_other_** .md files. (Of course I can fix remoting-artery.md in this PR)